recipe/meta.yaml

```diff
@@ -1,50 +1,51 @@
 {% set name = "pynndescent" %}
-{% set version = "0.5.4" %}
+{% set version = "0.5.10" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 221124cbad8e3cf3ed421a4089d80ac5a29d3215e76cb49effc1df887533d2a8
+  sha256: 5d5dc683c03ef55fe3ddf693859720ca18f85c6e6e5bb0b4f14870278d5288ad
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  # numba and llvmlite aren't available on s390x
+  skip: true  # [py<36 or s390x]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   host:
-    - python >=3.6
-    - setuptools
+    - python
     - pip
+    - setuptools
     - wheel
   run:
-    - python >=3.6
-    - numpy >=1.17
-    - numba >=0.51.2
+    - python
+    - importlib-metadata >=4.8.1  # [py<38]
+    - joblib >=0.11
     - llvmlite >=0.30
+    - numba >=0.51.2
+    - numpy >=1.17
     - scikit-learn >=0.18
     - scipy >=1.0
-    - joblib >=0.11
 
 test:
+  imports:
+    - pynndescent
   requires:
     - python
```

Comment:
python is in runtime already, so not needed here ?

Reply:
Removed